client/e2e/user-list.e2e-spec.ts

```diff
@@ -29,18 +29,15 @@ describe('angular-spark-lab', () => {
         expect(page.getUserTitle()).toEqual('User Name');
     });
 
-    it('should type something in filer name box and check that it returned correct element', () => {
+    it('should type something in filter name box and check that it returned correct element', () => {
         page.navigateTo();
         page.typeAName("Lynn");
         expect(page.getFirstUser()).toEqual("Lynn Ferguson is 25 years old");
     });
 
     it('should click on the age 27 times and return 3 elements then ', () => {
         page.navigateTo();
-        page.getUserByAge();
-        for (let i = 0; i < 27; i++) {
-            page.selectUpKey();
-        }
+        page.getUserByAge(27);
```

Comment:
Should the description's bit (line 38 just above) about "click on the age 27 times" change, since you don't seem to be doing that anymore? Or am I misunderstanding something?